### PR TITLE
Add logical/math operators to fix #739 

### LINKED ISSUE #739: [Kotlin] Add logical/math operators
# Description

Logic/math operators for Kotlin aren't presently on Code Thesaurus. This issue is to add that concept to the system.

# Directions

* On the home page, on the right side (the reference sheet side), choose the language and version, and then the concept.
* The page says the data doesn't exist, but clicking the arrow for the template will provide the template and file for you to use.
* Save that file in `/web/thesauruses/[language]/[version]/operators.json`.
* Fill out all of the `"code"` sections.
* Add in `"not-implemented": "true"` if the concept doesn't exist in the language. You can then delete the `"code"` line.
* Add in `"comment":` if you wish to explain something in the code
* Delete the `"name":` fields before your PR
* Save, commit your code, push to your fork, then make a PR.
* Watch the checks to make sure they all pass. If they don't, click on "Details" and it should tell you what to modify.

# Claiming This Issue

Please leave a comment below so I can assign you to it. This is to prevent multiple people from working on the same issue. 

Check out [CONTRIBUTING.md](https://github.com/codethesaurus/codethesaur.us/blob/main/CONTRIBUTING.md) for more information on how to contribute. 

Check out [https://docs.codethesaur.us/thesaurus/](https://docs.codethesaur.us/thesaurus/) for information on how Code Thesaurus works and how to add to the thesauruses.

# Need Help?

Leave a comment on here with your question, or feel free to ask [@geekygirlsarah](https://twitter.com/geekygirlsarah) on Twitter. You can also email coreteam@codethesaur.us too.

# Thanks!

We really value and appreciate the additions people make to Code Thesaurus, no matter how large or small. So thank you for taking the time to work on this!

### PR DESCRIPTION
**Pull request by Add00**
## What GitHub issue does this PR apply to?

 

Resolves #739

## What changed and why?
I've added the full set of arithmetic, logical, and bit-wise operators for the Kotlin language.

## Checklist

   

- [x] I claimed any associated issue(s) and they are not someone else's
- [x] I have looked at documentation to ensure I made any revisions correctly
- [x] I tested my changes locally to ensure they work
- [x] For language files, I have validated the edited files are valid JSON and data shows up correctly
- [x] For website code edits, I have added or edited any appropriate unit tests for my changes

## Any additional comments or things to be aware of while reviewing?

   
[Kotlin docs](https://kotlinlang.org/docs/home.html) were the primary source and instrumental in resolving this issue.

### CODE REVIEW

**Comment by Add00** [comment]
I didn't use docker to build the project, nor am I supper familiar with it. How should I best go about fixing the code to pass that test case?

**Comment by geekygirlsarah** [comment]
> I didn't use docker to build the project, nor am I supper familiar with it. How should I best go about fixing the code to pass that test case?

One of the reasons it's not a required check is that occasionally the build of that fails due to a library that sometimes breaks when the build happens. Nothing has changed with Docker, so that's fine. I've merged other PRs despite that failing.

File: web/thesauruses/kotlin/1.5/operators.json [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+      "greater_than": {
+        "name": "Greater than operator",
+        "code": "a > b"
+      },
+      "greater_than_or_equal_to": {
+        "name": "Greater than or equal to operator",
+        "code": "a >= b"
+      },
+      "null_coalescing": {
+        "name": "Elvis operator",
```
Comment (geekygirlsarah) [inline web/thesauruses/kotlin/1.5/operators.json]: The name doesn't technically show up, it's pulled in from the meta file. It'll still show up as a null-coalescing operator. You might want to clarify in the comment that's the case here (and also Kotlin calls it that)

**Review by geekygirlsarah — APPROVED** [review]
I can merge this in but don't know if you want to change the Elvis operator comment first. Let me know!

**Comment by Add00** [comment]
I changed the name to null coalescing and instead mentioned in the comment that it is also known as the Elvis operator

**Comment by geekygirlsarah** [comment]
You could have left it in there, just the name field is mostly for convenience of making the file here, not actually used.

**Review by geekygirlsarah — APPROVED** [review]

**Comment by geekygirlsarah** [comment]
You did, I didn't finish looking.

But thakn you for contributing! I'll merge it in.

Happy Hacktoberfest!

**Pull request merged**